### DISCUSSION TITLE
Zee Bangla

### DISCUSSION THREAD

**Discussion by krishnenduroy52** [iptv-org/community#859] [Q&A]
### Channel ID

ZeeBangla.in

### Stream URL

https://zeebangla.zee5.com/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19844
